src/V2/ApplicationV2/Sessions/OpcUaSession.cs

```diff
@@ -19,11 +18,21 @@
 using Opc.Ua.Client;
 using EndpointConfiguration = Opc.Ua.EndpointConfiguration;
 using TypeInfo = Opc.Ua.TypeInfo;
+using OMP.PlantConnectivity.OpcUA.Models.Alarms;
+using OMP.PlantConnectivity.OpcUA.Services.Alarms;
+using OMP.PlantConnectivity.OpcUA.Services.Subscriptions;
 
 namespace OMP.PlantConnectivity.OpcUA.Sessions
 {
     internal class OpcUaSession : IOpcUaSession
     {
+        #region [Constants]
+        private const uint SessionTimeoutInMs = 100000;
```

Comment:
TODO: These settings we should allow to be altered from outside on startup. Provide these defaults but be able to alter it from config

Reply:
Done